includes/collection/class-outbox.php

```diff
@@ -223,7 +223,7 @@ public static function get_activity( $outbox_item ) {
 		}
 
 		if ( 'Update' === $type ) {
-			$activity->set_updated( gmdate( 'Y-m-d H:i:s', strtotime( $outbox_item->post_modified ) ) );
+			$activity->set_updated( gmdate( 'Y-m-d\TH:i:s\Z', strtotime( $outbox_item->post_modified ) ) );
```

Comment:
Should we save that in a constant to make sure we use the same format everywhere?